### DISCUSSION TITLE
Get Ranked Data

### DISCUSSION THREAD

**Discussion by PyFlat-JR** [LeleDerGrasshalmi/FortniteEndpointsDocumentation#147] [Q&A] [answered]
Hello,
I am trying to get the rank progress using the API. However, I don't really understand how to do the authorization. Could someone help me? Thanks in advance!

**Comment by LeleDerGrasshalmi** [comment]
OAuth 2 - https://github.com/LeleDerGrasshalmi/FortniteEndpointsDocumentation/blob/main/EpicGames/AccountService/Authentication/Token.md

**Comment by PyFlat-JR** [comment]
Once I have done this, how can I use the token to get the rank?
I don't understand how i have to use this: rankings:{namespace}:playerprogress READ

**Reply by LeleDerGrasshalmi** [reply, marked as answer]
That is the permission required to use the api
